src/features/DebugSession.ts

```diff
@@ -57,7 +57,7 @@ export class DebugSessionFeature implements IFeature {
             if (versionDetails.edition.toLowerCase() === "core" &&
                 platformDetails.operatingSystem !== OperatingSystem.Windows) {
 
-                let msg = "PowerShell Core does not support attaching to a host process on non-Windows operating systems.";
+                let msg = "PowerShell Core does not support attaching to a host process except on Windows.";
```

Comment:
I'd go with "PowerShell Core only supports attaching to a host process on Windows.", seems to flow better that way